README.md

```diff
@@ -12,8 +12,137 @@ The enclosed application contains:
 
 This is a simple web application written with Django. It implements a single view which will return everything the user inputs. The templating engine disables the automatic XSS protection feature which sanitizes the input. In addition, the XSSDisableMiddleware disables the XSS protection feature implemented in modern web browser.
 
-## Deployment
+## Tutorial
 
-This application contains a build file for [circleci](https://circleci.com/) to deploy the vulnerable application to [Heroku](https://heroku.com). There are build jobs defined to do a dependency check for the python application using [safety](https://pypi.org/project/safety/) and a dynamic application security test using the [Crashtest Security Suite](https://crashtest-security.com).
+This application contains a build file for [CircleCI](https://circleci.com/) to deploy the vulnerable application to [Heroku](https://heroku.com). There are build jobs defined to do a dependency check for the python application using [safety](https://pypi.org/project/safety/) and a dynamic application security test using the [Crashtest Security Suite](https://crashtest-security.com).
 
-To run these jobs, they need to be added to the defined workflow.
\ No newline at end of file
+This application is used within workshops hold by [Crashtest Security](https://crashtest-security.com/janosch). This tutorial contains the steps to follow the workshop. In case you want to attend one of those workshops, let us know via [e-mail](mailto:info@crashtest-security.com).
+
+### Create Accounts
+
+- [GitHub](https://github.com): Create an account to get access to the source code of this example projcet
+- [CircleCI](https://circleci.com/signup/): Login using GitHub to grant CircleCI access to your GitHub projects
+- [Heroku](https://heroku.com): Create an account to deploy the example application there
+- [Crashtest Security](https://crashtest.cloud): Create an account te conduct a dynamic vulnerability scan
+
+### Fork the Repository
+
+To get access to the repository code, fork this repository by clicking the "Fork" button on the to right: https://github.com/Phylu/devsecops-example-heroku
```

Comment:
shouldn't the URL point to the crashtest-security repo instead of your private repository?